mindmeld/bot/webex_bot_server.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+
+"""
+This module contains the Webex Bot Server component.
+"""
+
+import logging
+import json
+from flask import request
+import requests
+from ciscosparkapi import CiscoSparkAPI
+
+
+class WebexBotServer:
+    """
+    A sample server class for Webex Teams integration with any MindMeld application
+    """
+
+    def __init__(self, app, webhook_id, access_token, conv):
+        self.app = app
+        self.webhook_id = webhook_id
+        self.access_token = access_token
+        self.conv = conv
+
+        self.logger = logging.getLogger(__name__)
+
+        if not self.webhook_id or not self.access_token:
+            raise Exception('webhook_id and bot_access_token are not set')
+
+        self.spark_api = CiscoSparkAPI(self.access_token)
+        self.ACCESS_TOKEN_WITH_BEARER = 'Bearer ' + self.access_token
+        self.CISCO_API_URL = 'https://api.ciscospark.com/v1'
+
+        @self.app.route('/', methods=['POST'])
+        def handle_message(self):
+            me = self.spark_api.people.me()
+            data = request.get_json()
+
+            for key in ['personId', 'id', 'roomId']:
+                if key not in data['data'].keys():
+                    return 'OK'
```

Comment:
Change this to a 400s http code and issue a better error message. Moreover, these three lines can be condensed to one line like this:
```
if len(data['data'].keys() & set(['personId', 'id', 'roomId'])) == len(['personId', 'id', 'roomId']):
    return 400, payload
```

Reply:
This condensation does not have the intended effect (logically it should though). So I have kept it the same for now.